pkg/api/dynamicmux.go

```diff
@@ -136,5 +139,7 @@ func (m *dynamicMux) reloadAPIs() {
 }
 
 func (m *dynamicMux) close() {
-	close(m.done)
+	m.closeOnce.Do(func() {
```

Comment:
It's rare to support reclosable in Easegress, so it's better to add a comment here.

Reply:
updated! please check